include/BALL/VIEW/RENDERING/RENDERERS/glRenderer.h

```diff
@@ -38,13 +38,13 @@
 #endif
 
 #ifdef BALL_OS_DARWIN
+        #include <OpenGL/gl.h>
 	#include <OpenGL/glu.h>
 #else
+        #include <GL/gl.h>
 	#include <GL/glu.h>
 #endif
 
-#include <QtGui/QOpenGLFunctions_1_3>
```

Comment:
I think this is more of a cosmetic issue. From my point of view you can just go ahead and merge.